### ISSUE TITLE
Some of properties are missing on index.d.ts

### ISSUE THREAD

**Issue by kokutaro** [jbetancur/react-data-table-component#395]
## Describe the bug
While using the react-data-table-component with TypeScript, I noticed following issue.
1. Property `conditionalRowStyles` is missing in `interface IDataTableProps<T>` 
2.  Property `conditionalCellStyles` in `interface IDataTableColumn<T>` should be `array`.

I suggest to correct the `index.d.ts` as below.

``` Typescript
export interface IDataTableProps<T> {
  ...
  conditionalRowStyles?:IDataTableConditionalRowStyles<T>[];
}
...
export interface IDataTableColumn<T> {
  ...
  conditionalCellStyles?: IDataTableConditionalCellStyles<T>[];
}
...
export interface IDataTableConditionalRowStyles<T> {
  when: (row: T) => boolean;
  style: CSSProperties;
}
```

**Comment by kokutaro** [comment]
I have created the pull request on #396

**Comment by jbetancur** [comment]
Fixed in #396

**Comment by jbetancur** [comment]
Slated for 6.0.5 release

**Comment by jbetancur** [comment]
Now available in 6.0.4!
